catalog-rest-service/src/main/resources/json/schema/api/data/createChart.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$id": "https://open-metadata.org/schema/api/data/createChart.json",
+  "$id": "https://github.com/open-metadata/OpenMetadata/blob/main/catalog-rest-service/src/main/resources/json/schema/api/data/createChart.json",
```

Comment:
@mithmatt we did this on purpose. There is rewrite rule to redirect these schemas

Reply:
Good to know. I'm flipping the other ones to use open-metadata.org